mms/service_manager.py

```diff
@@ -98,7 +98,7 @@ def get_loaded_modelservices(self, model_names=None):
             for model_name in model_names
         }
 
-    def load_model(self, model_name, model_dir, manifest, model_service_class_def, gpu=None, batch_size=None):
+    def load_model(self, model_name, model_dir, manifest, model_service_class_def, gpu=None, batch_size=None, manifest_legacy=None):
```

Comment:
I would put manifest_legacy in the middle of the parameters right after manifest.  It groups the manifest arguments together and allows easier utilization of the optional arguments. Also for register_and_load_modules.